packages/node/test/handlers.test.ts

```diff
@@ -192,7 +192,9 @@ describe('tracingHandler', () => {
     expect(transaction.traceId).toEqual('12312012123120121231201212312012');
     expect(transaction.parentSpanId).toEqual('1121201211212012');
     expect(transaction.sampled).toEqual(false);
-    expect(transaction.metadata?.baggage).toBeUndefined();
+    expect(transaction.metadata?.baggage).toBeDefined();
+    expect(isBaggageEmpty(transaction.metadata?.baggage)).toBeTruthy();
+    expect(isBaggageMutable(transaction.metadata?.baggage)).toBeFalsy();
```

Comment:
I recommend doing .toBe(true/false) instead of .toBeTruthy()/.toBeFalsy() unless we actually wanna check truthy/falsyness.

Reply:
Good point, thanks! e2ca92a9da026675a82fc25efccf59d9e45639b7